.gitignore

```diff
@@ -15,7 +15,7 @@ vendor/
 *.cache
 
 # local env files
-.env*
```

Comment:
Shouldn't this already be covered by `.env*` in line 60? AFAIK git matches subdirectories by default